templates/jlink/jlinked-app.yaml

```diff
@@ -105,6 +105,17 @@ objects:
       to:
         kind: ImageStreamTag
         name: intermediate:latest
+    triggers:
+      - type: ConfigChange
+      - type: ImageChange
+        imageChange:
+          from:
+            kind: ImageStreamTag
+            name: ubi9-openjdk-${JDK_VERSION}-jlink:latest  # Output of stage-1 which serves as input to stage-2
+      - type: GitHub
+        github:
+          secret: Gvw2O0tXuaAUxleJ
```

Comment:
pLease have a quick investigation into whether we can replace this secret with a template parameter, and if it’s possible for that to default to a random string.

Reply:
See this example https://github.com/jboss-container-images/openjdk/blob/b2201a5f43ff161db8e29bea4f5807de278400cf/templates/openjdk-web-basic-s2i.json#L66